src/stories/Autocomplete/Hook/AdvancedParameters.stories.tsx

```diff
@@ -49,3 +50,21 @@ addHookStoryCode(
   `const args = ${stringifyWithDefaults(TermsWithGroupSuggestions.args)}`,
   termsWithGroupSuggestionsDescription
 );
+
+export const FilteredSuggestions = HooksTemplate.bind({});
+FilteredSuggestions.args = {
+  apiKey,
+  onSubmit,
+  advancedParameters: {
+    autocompleteParameters: {
+      filters: {
+        group_id: '1130',
+      },
+    },
```

Comment:
as a consumer of this lib, I'd find this to be confusing:

instead of:
```js
advancedParameters: {
    autocompleteParameters: {
      filters: {
        group_id: '1130',
      },
    },
```

I'd prefer:
```js
advancedParameters: {
  filters: {
    group_id: '1130',
  },
},
```

Reply:
Good point, that makes sense to me. Updated to just combine the entire advancedParameter object with autocompleteParameters.

Our SDK should filter out any irrelevant parameters. What do you think?

https://github.com/Constructor-io/constructorio-ui-autocomplete/blob/c68b5b5531aa75fdca44086006c330972919bb80/src/hooks/useDebouncedFetchSections.ts#L63-L65